### DISCUSSION TITLE
Apikey

### DISCUSSION THREAD

**Discussion by sition** [statamic/cms#11153] [Q&A]
Hello,

Using postman I'm trying to post to a statamic endpoint. (It's a test). I've been given an Apikey
but I can't find any information on how to pass it. I've tried:

Authorization Bearer ...
Authorization Basic ...
ApiKey ...
api-key
Api-Key
APIKey
APIkey
apikey
api_key
key
x-api-key
X-API-KEY

But I keep getting 403.

Regards,

Jos

**Comment by ryanmitchell** [comment]
Statamic doesn't come with API authentication so it sounds like you're accessing a custom set up. It would be best to check with the original developer on how they set up auth.

**Reply by sition** [reply]
Ok, thanks.
